java-checks/src/main/java/org/sonar/java/checks/spring/TransactionalMethodVisibilityCheck.java

```diff
@@ -30,6 +31,16 @@
 @Rule(key = "S2230")
 public class TransactionalMethodVisibilityCheck extends IssuableSubscriptionVisitor {
 
+  private static final List<String> proxyAnnotations = List.of(
+    "org.springframework.transaction.annotation.Transactional",
+    "org.springframework.scheduling.annotation.Async"
+  );
+
+  private static final Map<String, String> annShortName = Map.ofEntries(
+    Map.entry("org.springframework.transaction.annotation.Transactional", "@Transactional"),
+    Map.entry("org.springframework.scheduling.annotation.Async", "@Async")
+  );
```

Comment:
Can we use `Map.of` here?
```suggestion
  private static final Map<String, String> annShortName = Map.of(
    "org.springframework.transaction.annotation.Transactional", "@Transactional",
    "org.springframework.scheduling.annotation.Async", "@Async"
  );
```